### ISSUE TITLE
More useful impetus

### ISSUE THREAD

**Issue by therealpant** [TheCodex6824/ThaumicAugmentation#395]
Hi, as for me, the branch with the "taming" of the rift is a bit useless. We can collect and store the energy of the rift (impetus), but there is not much to spend it on. We have a Cannon and a Terraformer.
Ideas: 
"Pylon of emptiness" - consumes rift energy and increases the maximum vis in the chunk (x2 so that it is not completely imbued) and the speed of its recovery
"Void enchantments" - enchantments also applied on the altar, but they begin to require fault energy to work, well, for balance, you can make a fine if they are not fed with this energy
"Not I came up with the name" - a multiblock with a model similar to Impetus Matrix in which you can stuff a broken pearl and slowly restore it using rift energy (for automation, you can add interaction with a comparator and a funnel)

**Comment by TheCodex6824** [comment]
I agree, Impetus right now doesn't have many uses. We have some plans in the works for adding more Impetus uses (and they will be the next feature update, which I hopefully will get through the bug backlog and start working on very soon).

**Comment by KELETU66666** [comment]
i'm not an advertise but there has a mod provides extra function with impetus:
https://www.curseforge.com/minecraft/mc-mods/advanced-smelter
